### PR TITLE
Ccache 4.6.3 => 4.7.4

### PR DESCRIPTION
**Pull request by uberhacker**
Works properly:
- [x] armv7l
- [x] i686
- [x] x86_64

### CODE REVIEW

File: packages/ccache.rb [hunk 1]
```diff
@@ -3,41 +3,46 @@
...
       #{CREW_CMAKE_OPTIONS} \
       -DCMAKE_INSTALL_SYSCONFDIR=#{CREW_PREFIX}/etc \
       -DZSTD_FROM_INTERNET=ON \
       -DHIREDIS_FROM_INTERNET=ON \
       .."
       system 'mold -run ninja'
     end
+    @ccacheenv = <<~CCACHEEOF
+      # ccache configuration
+      if ! [[ $PATH == *"ccache/bin"* ]]; then
```
Comment (supechicken) [inline packages/ccache.rb:42]: Use `!=`?

Comment (uberhacker) [reply packages/ccache.rb:42]: That might work but this is a regex evaluation.

File: packages/ccache.rb [hunk 2]
```diff
@@ -3,41 +3,46 @@
...
+        export PATH="#{CREW_LIB_PREFIX}/ccache/bin:$PATH"
```
Comment (supechicken) [inline packages/ccache.rb:43]: `export` is not necessary on env files

File: packages/ccache.rb (hunk 1 above)
Comment (supechicken) [reply packages/ccache.rb:42]: This is globbing... (regex uses `~=` and `!=` as operator)

Comment (uberhacker) [reply packages/ccache.rb:42]: Right, I just tested with `!=` and it works.  Rebuilding binaries... :sleeping:

Comment (satmandu) [reply packages/ccache.rb:42]: Does using `ccache` make the rebuild go faster? 🤣

Comment (uberhacker) [reply packages/ccache.rb]: You asked too late.  Already did the rebuilds.  How do you use this anyway?

**Review by satmandu — APPROVED** [review]

File: packages/ccache.rb (hunk 1 above)
Comment (satmandu) [reply packages/ccache.rb]: You add the ccache dir to the path first, and then ccache intercepts calls to the compiler and maintains/checks a cache of compiled code for every compile request.

**Pull request merged**